src/components/CodeInput/inputItems.js

```diff
@@ -44,18 +43,16 @@ const InputItems = React.forwardRef((props, ref) => {
                 onKeyDown={onKeyDown}
                 onPaste={onPaste}
                 ref={inputRef}
-                id={inputId}
                 isActive={isActive}
             />
         );
     });
 
-    return <div>{inputs}</div>;
+    return <StyledInputItems>{inputs}</StyledInputItems>;
```

Comment:
I think we should not need this wrapper here, instead, we should use a `Fragment`